shared/cfg/guest-hw.cfg

```diff
@@ -70,8 +77,10 @@ variants:
         drive_format=virtio
         i440fx:
             cd_format=ide
-        q35:
+        q35, q35-with-bridge:
```

Comment:
I looked at virtmanager in F23 and it creates a root complex + pci-pci bridge. So I think it's indeed a standard and the default q35 should create it. I'm sorry for confusion and I say nothing against that part anymore. Can you please revert it?